config/prow/plugins.yaml

```diff
@@ -1100,6 +1100,20 @@ cherry_pick_unapproved:
 
     For details on the patch release process and schedule, see the [Patch Releases](https://k8s.io/releases/patch-releases) page.
 
+# TODO: enable when https://github.com/kubernetes/test-infra/pull/31286 merged
+# cherry_pick_approved:
+# - org: kubernetes
+#   repo: kubernetes
+#   approvers:
```

Comment:
Nit: let's leave a comment with URL where this list is coming from (e.g. OWNERS_ALIASES in k/k or [k/website](https://kubernetes.io/releases/release-managers/#release-managers))

Reply:
Yep, sure thing. Added a link.